src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -21,16 +21,21 @@
 				>
 					<summary-card
 						data-testid="bp-summary"
-						class="tw-relative lg:tw--mb-1.5 tw-z-1"
+						class="tw-relative lg:tw--mb-1.5"
 						:show-urgency-exp="showUrgencyExp"
 						:lenders="lenders"
 						:social-exp-enabled="socialExpEnabled"
-					/>
+					>
+						<template #sharebutton v-if="inPfp">
+							<!-- Share button for PFP loans -->
+							<social-share-button class="tw-block md:tw-hidden tw-mt-3" :loan="loan" :lender="lender" />
```

Comment:
Is this "first version" of sharing on BP tightly coupled to the `inPfp` state or could we make it available to the general public as well...

Reply:
The spec was just for PFP profiles. I will bring this up to the rest of the team. I see no reason why this couldn't be opened up to all profiles.